src/util.h

```diff
@@ -7,13 +7,25 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <stdio.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include "light_elf.h"
```

Comment:
Breaks with code style for includes. They're not sorted alphabetically, and `light_elf.h` is a local include, not a system one.